src/decomp_2d_constants.f90

```diff
@@ -100,6 +100,27 @@ module decomp_2d_constants
    integer, parameter, public :: PHYSICAL_IN_X = 1 ! Forward is input in X, output in Z
    integer, parameter, public :: PHYSICAL_IN_Z = 3 ! Forward is input in Z, output in X
 
+   !
+   ! Family of readers / writers
+   !
+   integer, parameter, public :: DECOMP_2D_IO_NONE = 0
+   integer, parameter, public :: DECOMP_2D_IO_MPI = 1
+   integer, parameter, public :: DECOMP_2D_IO_ADIOS2 = 2
+
+   !
+   ! Choice for IO operations
+   !
+   integer, parameter, public :: DECOMP_2D_WRITE_MODE = 1, &
+                                 DECOMP_2D_READ_MODE = 2, &
+                                 DECOMP_2D_APPEND_MODE = 3
+
+   integer, parameter, public :: DECOMP_2D_WRITE_DEFERRED = 1, &
+                                 DECOMP_2D_WRITE_SYNC = 2
+   !
+   ! Interpolation methods
+   !
+   integer, parameter, public :: DECOMP_2D_INTERP_BASIC = 0
```

Comment:
Is basic 2nd order or using the compact interpolation schemes? Could add a comment

Reply:
I think the method is order zero (closest neighbor). I will provide some comments.

Please note that the addition of the interpolation is connected with the removal of the parameters icoarse / nstat/  nvisu / nprobe.

Reply:
Fixed in https://github.com/2decomp-fft/2decomp-fft/pull/269/commits/4109f442f1a3c60bf7b87c9cc92ddca372e65981